### PR TITLE
 [13.0][FIX] shopfloor - api_key_demo not found

### PR DESCRIPTION
**Pull request by hparfr**
Can't call None.sudo()

### CODE REVIEW

File: shopfloor/services/service.py [hunk 1]
```diff
@@ -267,7 +267,10 @@ def _get_openapi_default_parameters(self):
         # because it's a demo key.
         demo_api_key = self.env.ref(
```
Comment (simahawk) [inline shopfloor/services/service.py]: isn't this useless as we don't have the key at all?
Shall we do something better? Either add the key back for demo only or trash this bits and put "PUT_YOUR_KEY_HERE" in value?

Comment (hparfr) [reply shopfloor/services/service.py]: IMO it's more related to base_rest, you can expect the same behavior in shopinvader as well. If something is needed to be done; it should be on base_rest.

I'm in favor to remove this demo_api_key from shopfloor;  I don't use this feature btw.

Comment (guewen) [reply shopfloor/services/service.py:268]: It's useful when testing the module with demo data

Comment (guewen) [reply shopfloor/services/service.py]: It's useful when testing the module with demo data for faster development. When not using demo data, we don't care if the key is empty in swagger

**Comment by guewen** [comment]
Travis fails because of pre-commit

File: shopfloor/services/service.py (hunk 1 above)
Comment (simahawk) [reply shopfloor/services/service.py]: alright, fine then.

**Comment by hparfr** [comment]
> Travis fails because of pre-commit

Fixed. It's now green

**Review by simahawk — APPROVED** [review]

**Comment by simahawk** [comment]
@hparfr please, include module name in commits next time :wink:

**Pull request closed without merging**